add short discussion on half-width

lheagy opened the pull request:
Added a short discussion on half-width to magnetics: interpretation. This closes #43 

![image](https://cloud.githubusercontent.com/assets/6361812/18818133/76898b10-8326-11e6-8f45-aef0a87d3b7c.png)

@fourndo: would you like to take a quick look before we merge?

fourndo commented:
@lheagy Suggesting a little re-shuffle of the content. Ideally would probably want to move general statements about Parametric and Inversion. But left those inside TKC for now

fourndo commented:
I m cool with the merge if you are

lheagy commented:
@fourndo : I renamed `Data Processing` --> `Direct Data Interpretation` and added a couple statements to the intro. I will merge this and we can iterate if need be.
